File: java/utility/MakeContext.java
package voxgig.bluefintecsecrsdk.utility;

import java.util.Map;

import voxgig.bluefintecsecrsdk.core.Context;

final class MakeContext {

  private MakeContext() {}

  static Context makeContext(Map<String, Object> ctxmap, Context basectx) {
    return new Context(ctxmap, basectx);
  }
}

File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// BluefinTecsEcr SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class BluefinTecsEcrMakeContext
{
    public static function call(array $ctxmap, ?BluefinTecsEcrContext $basectx): BluefinTecsEcrContext
    {
        return new BluefinTecsEcrContext($ctxmap, $basectx);
    }
}
